### DISCUSSION TITLE
Not able to specify the chat context

### DISCUSSION THREAD

**Discussion by MedjahdiKaram** [betalgo/openai#202] [Q&A]
**Describe the bug**
We are not able to specify the chat context

**Additional context**
The context should be added to be able to use the same chat

**Comment by kayhantolga** [comment]
Could you share related documentation about this, please?

**Comment by sannae** [comment]
Hi! First-time contributor here, please be understanding 😅 

This was first answered in [this Q&A discussion](https://github.com/betalgo/openai/discussions/130).

Apparently right now the OpenAI API doesn't support the chat historical context, see [this OpenAI Community discussion](https://community.openai.com/t/how-do-you-maintain-historical-context-in-repeat-api-calls/34395) for further details. 

The simplest way is to always include the whole conversation history - or a summary of it - in each new request (even [ChatGPT responded this way](https://community.openai.com/t/how-do-you-maintain-historical-context-in-repeat-api-calls/34395/14)😆 ), but I guess it's unsustainable as token consumption would skyrocket.

We'll probably have to wait for the API to offer session management to replicate the experience in [chat.openai.com](https://chat.openai.com/chat).
